CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Changes since v1.3.0-rc.1
   user, although it was ignored previously.
 - Fix `--sharens` failure on EL8.
 - Fix Harbor registry login failure.
+- Reset `$HOME` value if it is non-readable.  
```

Comment:
That sort of reads as implying that the $HOME value itself is unreadable instead of the directory it points to.  Also the meaning of "Reset" isn't clear.   Also, again it doesn't really communicate to the user how it affects them.  So I suggest instead
```
- Prevent container builds from failing when `$HOME` points to a non-readable directory.
```

Reply:
thanks for the suggestion. I have updated it.